### DISCUSSION TITLE
Datatable labels language change

### DISCUSSION THREAD

**Discussion by ArchangerOne** [rappasoft/laravel-livewire-tables#1064] [Q&A]
Hello, 

i'm maybe wrong but i have read all the v2 docs and there isn't one line about how to change datatable labels localisation. Translations are well published, but what tag/property to use french json by default instead of english ?

Regards

**Comment by lrljoe** [comment]
Can you clarify what you mean by labels please?  Or what the issue you're experiencing is?

If you're talking about the Column header/label, then you can reference a translation in the label like so:
```php
Column::make(__('app.FieldName'), 'name'),
```
or reference by name e.g.
```php
Column::make(__('Nearest Metro Station'), 'nearestMetro'),
```
Everything else **should** be referencing the localised value!
